recipes-apps/ovmenu-ng-skripts/files/download-igc.sh

```diff
@@ -1,16 +1,50 @@
 #!/bin/bash
 
 USB_PATH="/usb/usbstick/openvario/igc"
-IGC_PATH="/home/root/.xcsoar/logs"
-mkdir -p $USB_PATH
```

Comment:
Hm, I think you should not remove this `mkdir`. New USB sticks won't have `$USB_PATH` initially.